Tools/ros2/ardupilot_dds_tests/launch/micro_ros_agent.launch.py

```diff
@@ -0,0 +1,109 @@
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument
+from launch.actions import OpaqueFunction
+from launch.substitutions import LaunchConfiguration
+
+import launch_ros
+
+
+def launch_micro_ros_agent(context, *args, **kwargs):
+    """
+    Launch the micro_ros_agent node.
+    """
+    # ros arguments
+    micro_ros_agent_ns = LaunchConfiguration("micro_ros_agent_ns")\
+        .perform(context)
+
+    # common arguments
+    transport = LaunchConfiguration("transport").perform(context)
+    middleware = LaunchConfiguration("middleware").perform(context)
+    refs = LaunchConfiguration("refs").perform(context)
```

Comment:
Do you want to validate the refs file exists prior to passing it to the application?

Reply:
I haven't added validation to the any arguments. I guess that's a policy we might want to decide - whether to put validation in what's effectively a pass through / translation layer, add it over the top, or leave it to the underlying node to handle. Whatever we decide it should prob apply to all arguments so it's consistent.